docs/sources/administration/api-keys/index.md

```diff
@@ -182,6 +182,23 @@ curl --request GET --url http://localhost:3000/api/folders --header 'Authorizati
 
 ### Migrate API keys to Grafana service accounts in Terraform
 
+{{% admonition type="note" %}}
+The terraform resource `api_key` has been removed in v3.0.0 from the Grafana Terraform Provider. During your migration path, we suggest that you pin your terraform version to <v2.19.0. See [release notes](https://github.com/grafana/terraform-provider-grafana/releases/tag/v3.0.0) for more information.
+{{% /admonition %}}
+
+If you need to pin the version of terraform to v2.19.0
+
+```tf
+terraform {
+  required_providers {
+    grafana = {
+      source  = "grafana/grafana"
+      version = "2.19.0"
+    }
+  }
+}
+```
+
```

Comment:
- Use `{{<` and `>}}` instead of `{{%` and `%}}`.
  
  The former has the most consistent semantics.
  The latter is used for special behavior that isn't required with this shortcode.
  
- Prefer 'refer to' over 'see' and add additional context to the link text: https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#links-and-references

- Consistently refer to Grafana Terraform Provider throughout

- Clarify the version constraints.

- Use `hcl` for syntax highlighting. I'm not sure that `tf` is supported by https://prismjs.com/.


````suggestion
{{< admonition type="note" >}}
The terraform resource `api_key` is removed from the Grafana Terraform Provider in v3.0.0.
Before you migrate and remove the use of the resource, you should pin your terraform version to a version less-than or equal-to v2.19.0.
For more information, refer to the [Grafana Terraform Provider release notes](https://github.com/grafana/terraform-provider-grafana/releases/tag/v3.0.0).
{{< /admonition >}}

To pin the Grafana Terraform Provider to v2.19.0:

```hcl
terraform {
  required_providers {
    grafana = {
      source  = "grafana/grafana"
      version = "2.19.0"
    }
  }
}
```

````